plejd/types/DeviceRegistry.d.ts

```diff
@@ -19,3 +19,17 @@ export interface OutputDevice {
   version: string;
   uniqueId: string;
 }
+
+export type InputDevices = { [deviceIdAndOutput: string]: InputDevice };
+
+export interface InputDevice {
+  bleOutputAddress: number;
```

Comment:
bleInputAddress?
bleAddress?

Reply:
bleInputAddress I think is better. Else we should update all places that have bleOutputAddress as well.

Same as above, change will happen later in the week unless you fix at merge.

Thanks for the comment. Coded as profession in c a veeeery long time ago. Played in JavaScript when Netscape was still a thing and had just release support for it. The language has come a long way since ... :-)